alf/networks/networks.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Various concrete Networks."""
+
+import torch
+import torch.nn as nn
+
+import alf
+
+from .network import Network, wrap_as_network
+
+
+class LSTMCell(Network):
```

Comment:
Can we directly wrap the pytorch one? https://pytorch.org/docs/stable/generated/torch.nn.LSTMCell.html

Reply:
Haven't done this before. Not sure if this is a workaround https://github.com/sphinx-doc/sphinx/issues/3140